plugins/GSdx/GSHwHack.cpp

```diff
@@ -85,22 +85,22 @@ bool GSC_DBZBT3(const GSFrameInfo& fi, int& skip)
 		}
 		else if(fi.TME && (fi.FBP == 0x00000 || fi.FBP == 0x00e00 || fi.FBP == 0x01000) && fi.FPSM == PSM_PSMCT32 && fi.TPSM == PSM_PSMT8H)
 		{
-			if (Dx_only) { // Ought to be fine with blending accuracy (fbmask?)
+			if (Aggressive) { // Partially works on D3D. OpenGL has no such issues.
 				if(fi.FBMSK == 0x00000)
 				{
-					skip = 28; // outline
-				}
-				if(fi.FBMSK == 0x00FFFFFF)
-				{
-					skip = 1;
+					// Hack removes character outlines as well as black borders on the sides of the screen.
+					// In the past the hack was used to remove red vertical lines on the screen I believe but now that issue has been fixed.
+					// The outlines partially work so on D3D9 can cause one of the borders at the bottom to shake, this can be fixed by removing the outlines with Aggressive crc or both x and y offsets.
```

Comment:
`Without the hack one of the borders at the bottom of the screen can start shaking when using the Direct3D 9 renderer. This can be fixed by using the TC Offsets x and y hardware hacks.`

The line below this can be removed as far as I'm concerned.